### ISSUE TITLE
OnPremise Sentry - Oracle Database

### ISSUE THREAD

**Issue by karacago24** [getsentry/self-hosted#1305]
### Problem Statement

Hi Sentry Team,

I want to use sentry with onprem installation andi want to use Oracle/ MSSQL instead of postgresql as Sentry database. 
Is there any support for oracle or mssql.

Thanks,

**Comment by chadwhitacre** [comment]
No. We stopped supporting databases other than PostgreSQL in April, 2019:

https://github.com/getsentry/sentry/pull/12668 &larr; includes removal of Oracle and MSSql
https://github.com/getsentry/sentry/pull/12640
https://github.com/getsentry/sentry/pull/12642

Here's [a good statement of the reason why](https://forum.sentry.io/t/still-does-not-support-mysql/1483/4?u=chadwhitacre).

**Comment by karacago24** [comment]
Thanks for your quick reply. It was very helpful
